Dockerfile

```
@@ -0,0 +1,14 @@
+FROM openethereum/openethereum:v3.1.1
+
+USER root
+RUN apk update && apk add bash
+RUN apk add --no-cache jq
```

I'm not sure we want bash in the image, but if so:

```suggestion
RUN apk update
RUN apk add --no-cache jq bash
```